src/test/java/com/amazon/dlic/auth/http/jwt/HTTPJwtAuthenticatorTest.java

```diff
@@ -227,6 +227,38 @@ public void testRoles() throws Exception {
         Assert.assertEquals("Leonard McCoy", creds.getUsername());
         Assert.assertEquals(2, creds.getBackendRoles().size());
     }
+    @Test
+    public void testRolesPath() throws Exception {
+
+
+
+        Settings settings = Settings.builder()
+                .put("signing_key", BaseEncoding.base64().encode(secretKey))
+                .put("roles_path", "$[\"access\"][\"roles\"]")
```

Comment:
we should also add a test case when both the rolesKey and rolesPath are specified.

Reply:
ok, will add it

Reply:
and also when none of the above are specified

Reply:
I think we already have this test,` testRolesMissing()`

Reply:
I meant no rolesPath or rolesKey in the configuration

Reply:
oh i missed it, ok will add